### ISSUE TITLE
Discard changes when exiting edit mode

### ISSUE THREAD

**Issue by Gia90** [BijoySingh/Material-Notes-Android-App#61]
When editing a note, I couldn't find any way to exit without saving the changes.
I think the "X" button in the top left corner is meant for that, but even when I press it to exit, the changes are not discarded.

**Comment by BijoySingh** [comment]
Yeah, the x button is not meant for that... (and yes, i know the poor design choice of a cross, a tick and the ancroid back-button all saving)
In fact I am thinking of removing the top row all together, to increase the space on the screen meant for typing. BUT, when I do that, it will be accompanied with Undo also, which should solve your problem :)
